Docs: what's new in enterprise 7.3 (#28472)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-3.md

### Auditing

Auditing tracks important changes to your Grafana instance to help you manage and mitigate suspicious activity and meet compliance requirements. Grafana logs events (as JSON) to file or directly to [loki](/oss/loki/).

Example of a login event:

```json
{
  "timestamp": "2020-10-22T10:18:00.838094347Z",
  "user": {
    "userId": 1,
    "orgId": 1,
    "isAnonymous": false
  },
  "action": "login-grafana",
  "result": {
    "statusType": "success",
    "statusCode": 200
  },
  "requestUri": "/login",
  "ipAddress": "127.0.0.1:41324",
  "userAgent": "Chrome/86.0.4240.111",
  "grafanaVersion": "7.3.0"
}
``` 

> For more details, see the [Auditing docs](/).

Reviewer: For more information, refer to the [Auditing docs](/).

Reviewer 2: @AgnesToulet , please add a link when your PR is merged.

> ### Datasource Usage Insights

Reviewer: Should be two words @oddlittlebird ?
"data source"

Reviewer 3: Yes, we use data source, not datasource.

Reviewer 2: Correct, per the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#data-source).

> Datasource usage insights is a new feature in Grafana Enterprise 7.3 that allows you to gain insight into how a datasource is being used and how well it works. There is a new tab in the datasource settings page called insights that will show you information about how the datasource has been used in the past 30 days.

Reviewer: Remove "is a new feature"
Data source should be two words?

> Insights per day:

Reviewer 4: Not sure I understand this part, do we have the insights 👇  for the last 30 days or is there a count looking 30 days back every day?

> - Queries last 30 days

Reviewer: Queries in the last 30 days

> - Errors last 30 days

Reviewer: Errors during the last 30 days

> - Average load duration last 30 days (ms)

Reviewer: Average load duration in the last 30 days (ms)

> ### SAML Improvements

Reviewer 2: Per the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#headings), avoid following one heading with another heading.

Please add at least a small intro sentence after the h3.

Reviewer 2: Also per the style guide, all headings should be in sentence case.

#### Single Logout

> SAML’s single logout feature allows users to log out from all applications associated with the current IdP session established via SAML SSO. Learn more in the [docs]({{<relref "../enterprise/saml/#single-logout">}}). 

Reviewer: SAML’s single logout capability allows users to log out from all applications associated with the current IdP session established via SAML SSO. For more information, refer to the [docs]({{<relref “../enterprise/saml/#single-logout”>}}).

> SAML’s single logout feature allows users to log out from all applications associated with the current IdP session established via SAML SSO. Learn more in the [docs]({{<relref "../enterprise/saml/#single-logout">}}). 

Reviewer 4: ```suggestion
SAML’s Single Logout (SLO) feature allows users to log out from all applications associated with the current IdP session established via SAML SSO. Learn more in the [SAML Single Logout docs]({{<relref "../enterprise/saml/#single-logout">}}). 
```

#### IdP-initiated Single-Sign-On

> IdP-initiated Single-Sign-On allows logins directly from the SAML identity provider (IdP). This feature is disabled by default for security reasons. Learn more in the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 

Reviewer: It is disabled by default for security reasons. For more information, refer to the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}).

> #### IdP-initiated Single-Sign-On
> 
> IdP-initiated Single-Sign-On allows logins directly from the SAML identity provider (IdP). This feature is disabled by default for security reasons. Learn more in the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 

Reviewer 4: ```suggestion
#### IdP-initiated Single Sign-On

IdP-initiated Single Sign-On (SSO) allows logins directly from the SAML identity provider (IdP). This feature is disabled by default for security reasons. Learn more in the [IdP initiated single sign-on docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 
```

> IdP-initiated Single-Sign-On allows logins directly from the SAML identity provider (IdP). This feature is disabled by default for security reasons. Learn more in the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 

Reviewer 3: ```suggestion
IdP-initiated Single-Sign-On allows to log in directly from the SAML identity provider (IdP). This feature is disabled by default for security reasons. Learn more in the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 
```

### Auditing

Auditing tracks important changes to your Grafana instance to help you manage and mitigate suspicious activity and meet compliance requirements. Grafana logs events (as JSON) to file or directly to [loki](/oss/loki/).

Example of a login event:

```json
{
  "timestamp": "2020-10-22T10:18:00.838094347Z",
  "user": {
    "userId": 1,
    "orgId": 1,
    "isAnonymous": false
  },
  "action": "login-grafana",
  "result": {
    "statusType": "success",
    "statusCode": 200
  },
  "requestUri": "/login",
  "ipAddress": "127.0.0.1:41324",
  "userAgent": "Chrome/86.0.4240.111",
  "grafanaVersion": "7.3.0"
}
``` 

For more details, see the [Auditing docs](/).

### Data source usage insights

Data source usage insights allows you to gain insight into how a data source is being used and how well it works. There is a new tab in the data source settings page called insights that will show you information about how the data source has been used in the past 30 days.

Insights:

- Queries per day
- Errors per day
- Average load duration per day (ms)

> ### SAML Single logout
> 
> SAML’s single logout (SLO) capability allows users to log out from all applications associated with the current IdP session established via SAML SSO. For more information, refer to the [docs]({{<relref "../enterprise/saml/#single-logout">}}). 
> 
> ### SAML IdP-initiated single sign on
> 
> IdP-initiated single sign on (SSO) allows the user to log in directly from the SAML identity provider (IdP). It is disabled by default for security reasons. For more information, refer to the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 

Reviewer 4: Suggesting shifting logout and sign on, because we define IdP in sign on. Alternatively write out identity provider for SLO as well.

Decapitalized the S in single logout header, as we're not stylizing it in the SSO header.

```suggestion
### SAML IdP-initiated single sign on

IdP-initiated single sign on (SSO) allows the user to log in directly from the SAML identity provider (IdP). It is disabled by default for security reasons. For more information, refer to the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 

### SAML single logout

SAML’s single logout (SLO) capability allows users to log out from all applications associated with the current IdP session established via SAML SSO. For more information, refer to the [docs]({{<relref "../enterprise/saml/#single-logout">}}). 
```

```suggestion
### SAML single logout

SAML’s single logout (SLO) capability allows users to log out from all applications associated with the current identity provider (IdP) session established via SAML SSO. For more information, refer to the [docs]({{<relref "../enterprise/saml/#single-logout">}}). 

### SAML IdP-initiated single sign on

IdP-initiated single sign on (SSO) allows the user to log in directly from the SAML identity provider (IdP). It is disabled by default for security reasons. For more information, refer to the [docs]({{<relref "../enterprise/saml/#idp-initiated-single-sign-on-sso">}}). 
```
